node/src/actors/session/handlers.rs

```diff
@@ -571,6 +581,7 @@ fn session_last_beacon_inbound(
         ..
     }: CheckpointBeacon,
 ) {
+    // TODO: LastBeacon on inbound peers?
```

Comment:
This is the function that handle the LastBeacon messages **from** inbound peers

Reply:
```rust
} else if chain_beacon.checkpoint == received_checkpoint {
                        info!("Our chain is on par with our peer's",);
                        // Create a last_beacon message
                        let last_beacon_msg = WitnetMessage::build_last_beacon(act.magic_number, chain_beacon);
                        // Write a last_beacon message in session
                        act.send_message(last_beacon_msg);
}
```
In this part it is already not necessary to respond with another last beacon, because synchronizing process is done through consensus